arches/app/search/components/standard_search_view.py

```diff
@@ -209,6 +209,13 @@ def handle_search_results_query(
         sorted_query_obj = search_filter_factory.create_search_query_dict(
             list(self.request.GET.items()) + list(self.request.POST.items())
         )
+
```

Comment:
It seems like folks who use a different search view would miss out on this functionality -- I understand the simplicity of not having `sort-by` and `sort-order` as json, however is there a way in which this logic could live within the search component `sort_results.py` file instead? That would make it more portable